docker/dockerfiles/Dockerfile

```diff
@@ -61,6 +61,7 @@ RUN mkdir -m 777 -p ${WORKBENCH_PUBLIC_DIR} && chown -R ${USER_NAME} ${WORKBENCH
 
 RUN apt update && \
     apt install -y --no-install-recommends \
+    curl \
```

Comment:
Why do you need curl inside docker?

Reply:
Because we are using curl here: https://github.com/openvinotoolkit/workbench/pull/97/files/7a5ea4b0a8ac7120a7aafbc8e941c46ceeb0b537#diff-6b30e5742c1955d7e441d44083db66fc1ac5ee49279eee3404a255f5ed9a061aR95

Reply:
You should check it with legal